rre-core/src/main/java/io/sease/rre/core/domain/metrics/MetricClassConfigurationManager.java

```diff
@@ -0,0 +1,90 @@
+package io.sease.rre.core.domain.metrics;
+
+import java.math.BigDecimal;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Singleton utility class for instantiating the metric class manager,
+ * and managing metric configuration details.
+ *
+ * @author Matt Pearce (mpearce@opensourceconnections.com)
+ */
+public class MetricClassConfigurationManager {
+
+    private static MetricClassConfigurationManager instance;
+
+    private BigDecimal defaultMaximumGrade = BigDecimal.valueOf(3);
+    private BigDecimal defaultMissingGrade = BigDecimal.valueOf(2);
+
+    private MetricClassConfigurationManager() {
+        // Private constructor
+    }
+
+    public static MetricClassConfigurationManager getInstance() {
```

Comment:
The constructor is empty so we could do 

`private final static MetricClassConfigurationManager INSTANCE = new MetricClassConfigurationManager();`

what do you think?

Reply:
Sure, I've just made that change. My thinking was to not create before it is required, but it is used at the very beginning, and is not a big class either, so it's a sensible change to make.